api4/channel_test.go

```diff
@@ -2347,7 +2347,7 @@ func TestUpdateChannelScheme(t *testing.T) {
 
 	th.App.SetPhase2PermissionsMigrationStatus(true)
 
-	team := &model.Team{
+	team, err := th.SystemAdminClient.CreateTeam(&model.Team{
```

Comment:
This is unrelated, but caused me some grief due to unhandled errors in a unit testing configuration.